internal/helmdeployer/deployer.go

```diff
@@ -428,7 +428,7 @@ func (h *Helm) install(bundleID string, manifest *manifest.Manifest, chart *char
 
 	u := action.NewUpgrade(&cfg)
 	u.Adopt = true
-	u.EnableDNS = true
+	u.EnableDNS = !options.DisableDNS
```

Comment:
Could we perhaps automate testing for this? Either through unit tests, checking that Helm is called with the right arguments, or by means of e2e tests maybe to validate actual deployments? How do you see it?